pipelines/build-deploy-test-CI-WL.yml

```diff
@@ -2786,6 +2788,55 @@ jobs:
     input_mapping: {acceptance-tests-repo: acceptance-tests-master,
                     batch-runner-repo: qid-batch-runner-master}
 
+- name: "CI Nightly Regression ATs"
+  serial: true
+  serial_groups: [ci-deploy-infrastructure,
+                  ci-acceptance-tests,
+                  action-scheduler-build,
+                  action-scheduler-deploy,
+                  action-worker-build,
+                  action-worker-deploy,
+                  case-api-build,
+                  case-api-deploy,
+                  case-processor-build,
+                  case-processor-deploy,
+                  uac-qid-service-build,
+                  uac-qid-service-deploy,
+                  pubsub-adapter-build,
+                  pubsub-adapter-deploy,
+                  print-file-service-build,
+                  print-file-service-deploy,
+                  fieldwork-adapter-build,
+                  fieldwork-adapter-deploy,
+                  notify-processor-build,
+                  notify-processor-deploy,
+                  notify-stub-build,
+                  notify-stub-deploy,
+                  exception-manager-build,
+                  exception-manager-deploy,
+                  toolbox-build,
+                  toolbox-deploy,
+                  database-monitor-deploy,
+                  rabbitmonitor-deploy,
+                  regional-counts-deploy]
+  plan:
+  - get: every-midnight
+    trigger: true
+    passed: ["CI Reset DB"]
```

Comment:
I think this job needs to also get and `passed:` on all the services/deployment jobs like the acceptance tests do but without the triggers. We need that to give us the audit trail in the pipeline of what commit it ran with.

Reply:
Very good point 👍